### PR TITLE
Update ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by ychoma**
Changing the order first 
Description, then steps to reproduce, then actual behavior, then expected behavior

### CODE REVIEW

**Comment by emanoelxavier** [comment]
I agree, something like: 

### Additional Information

#### Found in Version and Build

#### Environment Details

#### Validated in Version and Build

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -6,7 +8,14 @@
 
 ### Additional Information
 
+### Environment
+- OSC version/build: 
+- Cloud provider info:
+- Plugin info:
+- Others:
```
Comment (emanoelxavier) [inline ISSUE_TEMPLATE.md]: Others -> Other

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -6,7 +8,14 @@
...
+
 ### Status
 - [ ] Reproduced
 - [ ] In Progress
-- [ ] Validated
\ No newline at end of file
+- [ ] Validated     =>    Verified with build:
```
Comment (emanoelxavier) [inline ISSUE_TEMPLATE.md]: Verified with build: -> Verified in Version/Build:

File: ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -6,7 +8,14 @@
 
 ### Additional Information
 
+### Environment
+- OSC version/build:
```
Comment (emanoelxavier) [inline ISSUE_TEMPLATE.md]: Follow the capitalization used in other places: OSC Version/Build, Cloud Provider Info, Plugin Info

Comment (arvindn05) [inline ISSUE_TEMPLATE.md]: fixed

File: ISSUE_TEMPLATE.md (hunk 1 above)
Comment (arvindn05) [inline ISSUE_TEMPLATE.md]: fixed

File: ISSUE_TEMPLATE.md (hunk 2 above)
Comment (arvindn05) [inline ISSUE_TEMPLATE.md]: fixed

**Comment by arvindn05** [comment]
sorry merged the template before emanoels comments. But i updated the template in commit 470e73aef85a06c6e2c33b483679f3eb27376a81

**Pull request merged**